app/lib/frontend/templates.dart

```diff
@@ -132,7 +132,7 @@ class TemplateService {
     }
 
     final List<Map> dependencies = analysis
-        .getDependencies()
+        .getTransientDependencies()
```

Comment:
Wouldn't `transitive` be better than `transient`?

Reply:
ah, my bad, good catch